models/Tool/UUID/Dao.php

```diff
@@ -80,7 +80,7 @@ public function delete()
      */
     public function getByUuid($uuid)
     {
-        $data = $this->db->fetchAssociative('SELECT * FROM ' . self::TABLE_NAME ." where uuid='" . $uuid . "'");
+        $data = $this->db->fetchAssociative('SELECT * FROM ' . self::TABLE_NAME . ' where uuid= ?', [$uuid]);
```

Comment:
```suggestion
        $data = $this->db->fetchAssociative('SELECT * FROM ' . self::TABLE_NAME . ' where uuid= ?', [$this->db->quote($uuid)]);
```

Reply:
We should use `$this->db->quote()` here :)

Reply:
I suggest supplying the third param with a type instead `[Types::STRING]`:

> https://www.doctrine-project.org/projects/doctrine-dbal/en/2.13/reference/security.html#security

> Besides binding parameters you can also pass the type of the variable. This allows Doctrine or the underlying vendor to not only escape but also cast the value to the correct type. See the docs on querying and DQL in the respective chapters for more information.

Reply:
Why? Is a prepared statement not enough?